key.ui/src/main/java/de/uka/ilkd/key/gui/smt/TacletTranslationSelection.java

```diff
@@ -16,6 +17,120 @@
 import de.uka.ilkd.key.taclettranslation.assumptions.SupportedTaclets.TreeItem;
 import de.uka.ilkd.key.taclettranslation.assumptions.SupportedTaclets.TreeItem.SelectionMode;
 
+interface InfoListener {
+    void eventShowInfo(TreeItem item, TreeNode node);
+}
+
+
+public class TacletTranslationSelection {
```

Comment:
What does this do? Maybe a file you added by mistake?

Reply:
I do not know. Git blame goes back to the 2000s. Maybe I just touched it to adapt this to the new interfaces. 

I delete it.